hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1382,6 +1382,12 @@ public void rename(final Path source, final Path destination,
     } while (shouldContinue);
   }
 
+  private Boolean isCreateOnBlobEndpoint() {
```

Comment:
Use of Constant values for both true and config name

Reply:
done